Tests/SwiftDocCUtilitiesTests/ConvertActionTests.swift

```diff
@@ -1664,6 +1664,89 @@ class ConvertActionTests: XCTestCase {
         }
     }
     
+    func testConvertActionProducesDeterministicOutput() throws {
+        // Pretty printing the output JSON also enables sorting keys during encoding
+        // which is required for testing if the conversion output is deterministic.
+        let priorPrettyPrintValue = shouldPrettyPrintOutputJSON
+        shouldPrettyPrintOutputJSON = true
+        defer {
+            shouldPrettyPrintOutputJSON = priorPrettyPrintValue
+        }
```

Comment:
When running tests in parallel, I think this will affect other documentation builds (i.e., they might get `shouldPrettyPrintOutputJSON = yes`). One way to work around this I think would be to decode the JSON on disk below, then re-encode them using an encoder that has pretty formatting enabled.

Reply:
I added an in-source comment to clarify this but, even when run in parallel, only one test is being executed per test-process. So because we’re modifying this value in-process (instead of in the environment) there isn’t concern of this affecting other tests when run in parallel.